### PR TITLE
Test Cases for DataUtilities.calculateColumnTotal

### PR DESCRIPTION
**Pull request by bryson189**
Test cases were made for equivalence classes and edge cases.

Please review as some test cases may be overlapping/unnecessary.

### CODE REVIEW

File: Tests/CalculateColumnTotalTest.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+	//TODO
+	@Test
+	public void nullInputTest(){
+		try{
+			double result = DataUtilities.calculateColumnTotal(null, 0);
+			assertEquals(result, 0, .000000001d);
+		}catch(NullPointerException e){
+			fail("Expected 0 but null pointer exception caught instead.");
+		}
+	}
```
Comment (enochtsang) [inline Tests/CalculateColumnTotalTest.java]: Refer to PR #3 line 88 for testing for exceptions

**Comment by kkcmah** [comment]
I think the expected values and actual values in your assertEquals() calls are switched.
It should be assertEquals(expected, actual).

**Comment by bryson189** [comment]
Expected and actual values have been switched and the testing for exceptions have been changed to match line 88 in PR#3.

**Comment by bryson189** [comment]
Kelby's suggestions have been added: Added default timeout and assertion messages with every test case
